### ISSUE TITLE
 Edit index.js file source

### ISSUE THREAD

**Issue by reujea** [pzmosquito/eslint-import-resolver-vite#16]
Request to modify the source of line index.js:55.
It is good for path.sep to indicate the seperator for each OS, but the purpose of that line is to split the path written by the developer.
Developers can use seperators consistently or mixedly, and since we have a general understanding of path syntax, we request modifications as follows.

```
// existing syntax
const pathParts = actualSource.split(path.sep);

// Modification Syntax
let pathParts = actualSource.split("/");
if(pathParts.length == 1) pathParts = actualSource.split("\\");
```

**Comment by pzmosquito** [comment]
Thanks for the suggestion. I'll add it to the next release and publish it soon.

**Comment by pzmosquito** [comment]
released version [1.3.2](https://github.com/pzmosquito/eslint-import-resolver-vite/releases/tag/1.3.2)
